saleor/graphql/checkout/tests/deprecated/test_checkout_complete.py

```diff
@@ -115,6 +116,7 @@ def test_checkout_complete(
     variables = {"checkoutId": checkout_id, "redirectUrl": redirect_url}
     response = user_api_client.post_graphql(MUTATION_CHECKOUT_COMPLETE, variables)
 
+    flush_post_commit_hooks()
```

Comment:
`with django_capture_on_commit_callbacks(execute=True):` would be better, as `flush_post_commit_hooks()` doesn't handle well a rollback. (IMHO this helper should be fully drop from our core, as AFAIK it does the same what `django_capture_on_commit_callbacks`)